README.md

```diff
@@ -155,7 +155,7 @@ static camera_config_t camera_config = {
 
     .jpeg_quality = 12, //0-63 lower number means higher quality
     .fb_count = 1, //if more than one, i2s runs in continuous mode. Use only with JPEG
-    .grab_mode = CAMERA_GRAB_WHEN_EMPTY//CAMERA_GRAB_LATEST. Sets when buffers should be filled
+    // .grab_mode = CAMERA_GRAB_WHEN_EMPTY//CAMERA_GRAB_LATEST. Sets when buffers should be filled
```

Comment:
what is the reason for commenting out this line?

Reply:
if it is not commented out, you get this error when compiling:

```
...
Compiling .pio/build/esp32cam/FrameworkArduino/Esp.cpp.o
Compiling .pio/build/esp32cam/FrameworkArduino/FunctionalInterrupt.cpp.o
Compiling .pio/build/esp32cam/FrameworkArduino/HardwareSerial.cpp.o
Compiling .pio/build/esp32cam/FrameworkArduino/IPAddress.cpp.o
Compiling .pio/build/esp32cam/FrameworkArduino/IPv6Address.cpp.o
In file included from src/main.cpp:14:0:
src/cameraInit.h:53:18: error: 'CAMERA_GRAB_WHEN_EMPTY' was not declared in this scope
     .grab_mode = CAMERA_GRAB_WHEN_EMPTY // CAMERA_GRAB_LATEST. Sets when buffers should be filled
                  ^
src/cameraInit.h:54:1: error: 'camera_config_t' has no non-static data member named 'grab_mode'
 };
 ^
*** [.pio/build/esp32cam/src/main.cpp.o] Error 1
============================================================================================ [FAILED] Took 1.34 seconds ============================================================================================
make: *** [Makefile:19: build] Error 1
```

Reply:
the item is in the struct [here](https://github.com/espressif/esp32-camera/blob/master/driver/include/esp_camera.h#L146) maybe PIO is using the wrong version?

Reply:
Thats so odd, i dont have that on mine 
![image](https://user-images.githubusercontent.com/31870999/190246900-32173ef4-970b-4504-a8eb-832c156ee3bc.png)
 and my pio is set to use the latest version
```
...
lib_deps =                              ; External libraries
  esp32-camera                          ; https://github.com/espressif/esp32-camera
...
```

Reply:
Check the pio registry, its out of date: https://registry.platformio.org/libraries/espressif/esp32-camera